src/styles/index.scss

```diff
@@ -1,3 +1,154 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the `Roboto` font is correctly loaded in your HTML file, as it is used extensively throughout the styles.